simulation/test_transfer.cpp

```diff
@@ -218,7 +218,8 @@ TORRENT_TEST(no_proxy_utp_ipv6)
 {
 	using namespace lt;
 	run_test(
-		[](lt::session&, lt::session&) {},
+		[](lt::session& ses0, lt::session& ses1)
+		{ utp_only(ses0); utp_only(ses1); },
```

Comment:
utp is blocked by default by the test, so both sessions need utp re-enabled for any uTP test.